src/main/java/com/sda/microblogging/controller/UserController.java

```diff
@@ -156,7 +149,7 @@ public ResponseEntity updatePassword(@NotNull @RequestParam int userId, @Valid @
     @ApiOperation(value = "Update user privacy", notes = "Update user privacy")
     @ResponseStatus(HttpStatus.OK)
     @PatchMapping(path = "/updateIsPrivate/{isPrivate}")
-    public ResponseEntity updatePrivacy(@NotNull @PathVariable boolean isPrivate, @RequestBody UserLoginDTO userLoginDTO){
+    public ResponseEntity updatePrivacy(@NotNull @PathVariable boolean isPrivate, @RequestBody UserLoginDTO userLoginDTO) {
         //TODO below shall be changed after Security
         Optional<User> updatingUser = userService.findUserByEmail(userLoginDTO.getEmail());
```

Comment:
Since the service method throws an exception if the user is not found, we won't get an optional, we will always have a valid user if the user or an exception: 

```
public User updateUserPrivacy(@NotBlank String userEmail, boolean isPrivate) {
        User user = userRepository.findByEmail(userEmail).orElseThrow(UserNotFoundException::new);
        user.setPrivate(isPrivate);
        return userRepository.save(user);
}
```

Reply:
As there are no security now I temporarily have all this: UserLoginDTO and checks..